src/app/pages/OpenWalletPage/Features/ImportAccountsSelectionModal/index.tsx

```diff
@@ -44,32 +49,71 @@ function ImportAccountsSelector({ accounts }: ImportAccountsSelectorSelectorProp
 
 interface ImportAccountsSelectionModalProps {
   abort: () => void
-  type: 'mnemonic' | 'ledger'
+  type: WalletType.Mnemonic | WalletType.Ledger
 }
 
 export function ImportAccountsSelectionModal(props: ImportAccountsSelectionModalProps) {
   const { t } = useTranslation()
+  const size = useContext(ResponsiveContext)
   const importAccounts = useSelector(selectImportAccounts)
+  const accounts = useSelector(selectImportAccountsOnCurrentPage)
+  const pageNum = useSelector(selectImportAccountsPageNumber)
   const error = importAccounts.error
   const selectedAccounts = useSelector(selectSelectedAccounts)
   const dispatch = useDispatch()
 
   const openAccounts = () => {
     dispatch(
-      props.type === 'ledger'
+      props.type === WalletType.Ledger
         ? walletActions.openWalletsFromLedger()
         : walletActions.openWalletFromMnemonic(),
     )
   }
-  const cancelDisabled = importAccounts.step === ImportAccountsStep.Done || error ? false : true
-  const confirmDisabled = importAccounts.step !== ImportAccountsStep.Done || selectedAccounts.length === 0
+
+  const isBusyImporting = importAccounts.step !== ImportAccountsStep.Idle
+  const cancelDisabled = isBusyImporting && !error
+  const confirmDisabled = isBusyImporting || selectedAccounts.length === 0
+
+  const canGoPrev = !isBusyImporting && !!pageNum && !error
+
+  const onPrev = () => {
+    dispatch(importAccountsActions.setPage(pageNum - 1))
+  }
+
+  const canGoNext = !isBusyImporting && pageNum !== numberOfAccountPages - 1 && !error
```

Comment:
```ts
const canGoPrev = pageNum > 0 && !isBusyImporting && !error
const canGoNext = pageNum < numberOfPages && !isBusyImporting && !error
```
would be more readable

Reply:
OK, updated. (Except that we need `< numberOfPages - 1`.)